### DISCUSSION TITLE
[Feature] Allow cleanup of all "OFFLINE PATHS" files?

### DISCUSSION THREAD

**Discussion by briandipalma** [immich-app/immich#4594] [Feature Request]
### The feature

I can't find them in the Immich web GUI so I'm guessing they were deleted manually and Immich can't provide a zip of all the pictures? At that stage surely we should be able to just mass delete these files from the database?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by aviv926** [comment]
That's something I'm trying to explore as well
The suggestions found in this discord discussion failed to help me resolve the issue

https://discord.com/channels/979116623879368755/1166371286528835676

**Comment by briandipalma** [comment]
OK, I went through the list of files. Some are located in the upload directory e.g.

```
/usr/src/app/upload/upload/5e7dfe53-e21a-4ab6-bff8-48c43a5e7ab9/824667d0-7e29-4069-85ee-1239af84edda.JPG
```

those I couldn't see what I could do. They aren't in the upload directory and being in the upload folder I couldn't find them in the GUI. Totally flummoxed on these paths.

The other paths are in the library

```
/usr/src/app/upload/library/5e7dfe53-e21a-4ab6-bff8-48c43a5e7ab9/2009/02/20090203-105809.JPG
```

these I can find in the GUI and delete them manually. Re-running the repair job though doesn't clear them down. I'm guessing it just takes some time for the database to update?
